app/main/views.py

```diff
@@ -8,9 +8,17 @@
 def index():
     return render_template('main/index.html')
 
-
 @main.route('/about')
 def about():
     editable_html_obj = EditableHTML.get_editable_html('about')
     return render_template('main/about.html',
                            editable_html_obj=editable_html_obj)
+
+@main.route('/list-orgs')
+def list_orgs():
+    return render_template('main/list-orgs.html')
+
+@main.route('/<int:org_id>')
```

Comment:
do we possibly maybe want to have /organization/<int:org_id> instead?

Reply:
+1 on this. I would also advocate putting all organization stuff under its own blueprint since there will be a number of routes dedicated to that  (plus it would be easier to separate routes if we get to the point of having teacher accounts).

Reply:
Do you think we should put list-orgs under the organization, or under main? I can see arguments for both (esp w teacher accounts) but i agree, will move individual orgs to under the org route.